internal/zinc-benchmarks/src/test/scala/xsbt/ZincBenchmarkSpec.scala

```diff
@@ -15,50 +15,50 @@ class ZincBenchmarkSpec extends FunSuite {
     }
   }
 
-  test("Check that setup for small project is successful and zinc compiles") {
```

Comment:
Can we remove this test instead?

Reply:
ok.

Reply:
https://github.com/sbt/zinc/pull/640/commits/75b3a4b8f9888d7e28abc834f25db7c84f214f48